src/modules/dashboard/components/ExpenditurePage/Recipient/Recipient.tsx

```diff
@@ -0,0 +1,276 @@
+import { FieldArray, useField, useFormikContext } from 'formik';
+import { nanoid } from 'nanoid';
+import React from 'react';
+import { defineMessages, FormattedMessage } from 'react-intl';
+import classNames from 'classnames';
+
+import Button from '~core/Button';
+import {
+  FormSection,
+  Input,
+  InputLabel,
+  Select,
+  TokenSymbolSelector,
+} from '~core/Fields';
+import Icon from '~core/Icon';
+import { ItemDataType } from '~core/OmniPicker';
+import { Tooltip } from '~core/Popover';
+import { filterUserSelection } from '~core/SingleUserPicker';
+import UserAvatar from '~core/UserAvatar';
+import UserPickerWithSearch from '~core/UserPickerWithSearch';
+import { AnyUser } from '~data/index';
+import { Address } from '~types/index';
+import { Recipient as RecipientType } from '../Payments/types';
+import { tokensData } from './constants';
+
+import styles from './Recipient.css';
+
+const MSG = defineMessages({
+  defaultRecipientLabel: {
+    id: 'dashboard.Expenditures.Recipient.defaultRecipientLabel',
```

Comment:
```suggestion
    id: 'dashboard.ExpenditurePage.Recipient.defaultRecipientLabel',
```

Again ,this applies to all other message descriptor id's on this page